### PR TITLE
VACCINES CAUSE AUTISM

### PR DESCRIPTION
**Pull request by Iamgoofball**
no they dont

### CODE REVIEW

**Comment by tigercat2000** [comment]
lol

**Comment by coldud13** [comment]
merge immediately

**Comment by scrubmcnoob** [comment]
well everyone always said they wanted more realism in this game
finally a pr that will make me feel immersed

**Comment by Crazylemon64** [comment]
Nice meme 👌

File: code/modules/reagents/chemistry/reagents/water.dm [hunk 1]
```diff
@@ -228,6 +228,8 @@
 			if(D.GetDiseaseID() in data)
 				D.cure()
 		M.resistances |= data
+		if(prob(1))
+			M.adjustBrainLoss(100)
```
Comment (scrubmcnoob) [inline code/modules/reagents/chemistry/reagents/water.dm]: this should be made into 60 so they don't die
it be more realistic that way

**Comment by KasparoVy** [comment]
:ok_hand:

**Comment by uraniummeltdown** [comment]
m a n y s u c h c a s e s

**Comment by Fox-McCloud** [comment]
Memes

**Comment by Iamgoofball** [comment]
4 antivaxxers baited out
@SamHPurp
@Twinmold93 
@taukausanake 
@DragonFire101

**Comment by SamHPurp** [comment]
@Iamgoofball anti-memers

**Pull request closed without merging**